playground.html

```diff
@@ -1,2 +1,20 @@
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<style>
+.reference {
+  width: 600px;
+  height: 20px;
+}
+.basepair.a { fill: #188712; }
+.basepair.g { fill: #C45C16; }
+.basepair.c { fill: #0600F9; }
+.basepair.t { fill: #F70016; }
+.basepair.u { fill: #F70016; }
```

Comment:
Nothing to do for now, but it may be nice to use a CSS pre-processor at some pt, to make it much easier to skin the pileup viewer.